tests/integration/docker/docker-compose.yaml

```diff
@@ -32,6 +32,7 @@ services:
       - ELASTICSEARCH_URL=http://estunnel:9500
       - INDEX_PREFIX=search2
       - INDEX_PREFIX_DELIMITER=.
+      - INDEX_SUFFIX_DELIMITER=_
```

Comment:
Why?

Reply:
I don't know, I didn't investigate the black list closely, other than to notice that it is very large. I know we did discuss blacklisting in the last year in relation to search2. The question came up in relation to creating default indexes, and that we should not index types which are hidden in the Narrative. That is probably where that list came from.

What we should rather be doing is consulting some canonical listing of workspace types and using that for blacklisting. It think there is a flag for that when a type is added to a workspace. I haven't looked into our type registration process to know whether this flag is part of the type definition itself, or is specified as a flag when the type is added to the workspace. There could be room for error there, and for different definitions of hidden in different workspaces.

Reply:
Okay.  I think since we are excluding these from the default_search this should be fine.